parsec/core/config.py

```diff
@@ -122,18 +125,22 @@ def config_factory(
             "parsec://localhost:6777?no_ssl=true"
         )
 
+    if mountpoint_base_dir is None:
```

Comment:
This is an unrelated fix ?

Reply:
Yes, I noticed that the `mountpoint_base_dir` argument was unused.